### ISSUE TITLE
Missing newline, wrong indent in #if block.

### ISSUE THREAD

**Issue by jindraivanek** [fsprojects/fantomas#231]
1)
```
#if SOME_FLAG
printfn "Hello with flag"
#else
printfn "Hello without flag"    
#endif
```

Result

```
#if SOME_FLAGprintfn "Hello with flag"
#else
printfn "Hello without flag"
#endif
```

2)
```
do
#if SOME_FLAG
    printfn "Hello with flag"
#else
    printfn "Hello without flag"    
#endif
```

Result

```
do
#if SOME_FLAG printfn "Hello with flag"
#else
printfn "Hello without flag"
#endif
```

But

```
let x =
#if SOME_FLAG
    printfn "Hello with flag"
#else
    printfn "Hello without flag"    
#endif
```
works as expected.

**Comment by SteveGilham** [comment]
Under the same umbrella, in processing a class to place `member` declarations starting in column 9, a method inside an #if was not consistently updated.

**Comment by jindraivanek** [comment]
Example for `member` use:

```
type T = { x: int} with
#if SOME_FLAG
    static member X = ()
#else
    static member X = ()
#endif
```

output

```
type T =
    { x : int }

#if SOME_FLAGstatic member X = ()
#else
static member X = ()
#endif
```

**Comment by nojaf** [comment]
I believe this is fixed in 2.8.1. Could you verify this please @SteveGilham?

**Comment by SteveGilham** [comment]
I believe that the issues I marked in [#245](https://github.com/fsprojects/fantomas/issues/245) are indeed fixed in 2.8.1

**Comment by nojaf** [comment]
Hmm you are right, some indentation info is lost. Could you create a new issue for this. I'd like to follow it up there.
